test/e2e/_move.js

```diff
@@ -0,0 +1,383 @@
+ *
+ * @param {object} options - same options as can be passed to updateMovePage.fillInPersonalDetails
+ *
+ * @returns {undefined}
+ */
+export async function checkUpdatePersonalDetails(options) {
+  const { person } = t.ctx
+  const updateMovePage = await clickUpdateLink('personal_details')
+  const updatedFields = await updateMovePage.fillInPersonalDetails({}, options)
+  const updatedDetails = { ...person, ...updatedFields }
+
+  await updateMovePage.submitForm()
+
+  await moveDetailPage.checkPersonalDetails(updatedDetails)
+}
+
+/**
+ * Helper function change info on assessment answers page and confirm changes on move view page
+ *
+ * @param {string} page - page key
+ *
+ * @param {string} fillInMethod - updateMovePage method to use to fill in fields
+ *
+ * @param {string} checkMethod - updateMovePage method to use to check values
+ *
+ * @param {object} [options] - same options as can be passed to updateMovePage.fillInRiskInformation
+ *
+ * @param {boolean} [options.selectAll=true]
+ *
+ * @param {boolean} [options.fillInOptional=true]
+ *
+ * @returns {undefined}
+ */
+export async function checkUpdatePage(
+  page,
+  fillInMethod,
+  checkMethod,
+  { selectAll = true, fillInOptional = true } = {}
+) {
+  const { person } = t.ctx
+  const updateMovePage = await clickUpdateLink(page)
+  const updatedFields = await updateMovePage[fillInMethod]({
+    selectAll,
+    fillInOptional,
+  })
+  const updatedDetails = { ...person, ...updatedFields }
+
+  await updateMovePage.submitForm()
+
+  await moveDetailPage[checkMethod](updatedDetails)
+}
+
+/**
+ * Change risk information and confirm changes on move view page
+ *
+ * @param {object} [options] - same options as can be passed to updateMovePage.fillInRiskInformation
+ *
+ * @returns {undefined}
+ */
+export async function checkUpdateRiskInformation(options) {
+  await checkUpdatePage(
+    'risk',
+    'fillInRiskInformation',
+    'checkRiskInformation',
+    options
+  )
+}
+
+/**
+ * Change health information and confirm changes on move view page
+ *
+ * @param {object} [options] - same options as can be passed to updateMovePage.fillInHealthInformation
+ *
+ * @returns {undefined}
+ */
+export async function checkUpdateHealthInformation(options) {
+  await checkUpdatePage(
+    'health',
+    'fillInHealthInformation',
+    'checkHealthInformation',
+    options
+  )
+}
+
+/**
+ * Change court information and confirm changes on move view page
+ *
+ * @param {object} [options] - same options as can be passed to updateMovePage.fillInCourtInformation
+ *
+ * @returns {undefined}
+ */
+export async function checkUpdateCourtInformation(options) {
+  await checkUpdatePage(
+    'court',
+    'fillInCourtInformation',
+    'checkCourtInformation',
+    options
+  )
+}
+
+/**
+ * Check that PNC details are uneditable and displayed corectly on the personal details page
+ *
+ * @returns {undefined}
+ */
+export async function checkPoliceNationalComputerReadOnly() {
```

Comment:
I feel some of these should maybe live within the Page Objects? They're doing the same thing the methods there are.